plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/_core.py

```diff
@@ -225,6 +229,7 @@ def setup_ax_client(self, arguments: List[str]) -> AxClient:
         """Method to setup the Ax Client"""
         parameters: List[Dict[str, Any]] = []
         for key, value in self.ax_params.items():
+            key = normalize_key(key, str_to_replace="_", str_to_replace_with=".")
             param = OmegaConf.to_container(value, resolve=True)
```

Comment:
curious, why are you converting to a container here?
can't you just use the DictConfig ?

Reply:
Ax checks for types internally eg https://github.com/facebook/Ax/blob/f653de41fcf4c4a3bf53b15b02eb7730c9f6774c/ax/service/utils/instantiation.py#L80